README.md

```diff
@@ -1,2 +1,14 @@
-# lsstinstall
+lsstinstall
+==========
+
 Install eups distribution from eups.lsst.codes
+
+The tool (lsstinstall) is available in the bin folder,
+and can be used as is after cloning this repository.
```

Comment:
Is it meant to be used by cloning, by `curl` like `newinstall.sh`, or by conda-forge?  Or all three?  We should decide and make this explicit.

Reply:
I think we should point to conda-forge for general distribution and use cloning only for development. I think we should avoid curl... 

I try to clarify this in the README.